website/docs/support/known-issues.md

```diff
@@ -24,3 +24,5 @@ Below is a list of known Corso issues and limitations:
 
 * Exchange Calender Event instance exceptions (changes to a single event within a recurring series) aren't
 included in backup and restore.
+
+* Calendar backup and restore is only supported for calendars belonging to user's default calendar group.
```

Comment:
This is not true.  The query `https://graph.microsoft.com/v1.0/users/{{OwnerId}}/calendars` (which we use to enumerate calendars) returns all calendars independent of the group.  You can test this yourself, if wanted:
1. go to outlook, create a new calendar group
2. create a new calendar that only exists in that calendar group
3. (optional) give your new calendar an event
4. Query for all of your user's calendars: `https://graph.microsoft.com/{{version}}/users/{{OwnerId}}/calendars`

You should see the calendar owned by the non-default group in the results.

Reply:
Thanks! I didn't know this. I'll test it out. If it works, I'll close this PR